flocker/dockerplugin/test/test_api.py

```diff
@@ -174,6 +174,40 @@ def test_path(self):
                            u"Mountpoint": u"/flocker/{}".format(dataset_id)}))
         return d
 
+    def test_unknown_path(self):
+        """
+        ``/VolumeDriver.Path`` returns an error when asked for the mount path
+        of a non-existent volume.
+        """
+        name = u"myvol"
+        return self.assertResult(
+            b"POST", b"/VolumeDriver.Path",
+            {u"Name": name}, OK,
+            {u"Err": u"Volume not available.", u"Mountpoint": u""})
+
+    def test_non_local_path(self):
+        """
+        ``/VolumeDriver.Path`` returns an error when asked for the mount path
+        of a volume that is not mounted locally.
```

Comment:
:question: When might this occur? Maybe when `docker inspect`  is pointed at a flocker backed container that had been stopped and when the flocker volume had since been associated with a container on another node?